packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,386 @@
+
+const SECTION_HEADER_MARGIN_TOP = 32
+
+interface CategoryWithDapps {
+  id: string
+  name: string
+  fontColor: string
+  backgroundColor: string
+  dapps: Dapp[]
+}
+
+interface Dapp {
+  id: string
+  categoryId: string
+  iconUrl: string
+  name: string
+  description: string
+  dappUrl: string
+}
+
+interface DappProps {
+  dapp: Dapp
+  onPressDapp: (dapp: Dapp) => void
+}
+
+interface SectionData {
+  data: Dapp[]
+  category: CategoryWithDapps
+}
+
+export function DAppsExplorerScreen() {
+  const { t } = useTranslation()
+  const dappsListUrl = useSelector(dappsListApiUrlSelector)
+  const [isHelpDialogVisible, setHelpDialogVisible] = useState(false)
+  const [isBottomSheetVisible, setBottomSheetVisible] = useState(false)
+  const [dappSelected, setDappSelected] = useState<Dapp>()
+  const dispatch = useDispatch()
+
+  const {
+    loading,
+    error,
+    result,
+  }: {
+    loading: boolean
+    error: Error | undefined
+    result: CategoryWithDapps[] | undefined
+  } = useAsync(
+    async () => {
+      if (!dappsListUrl) {
+        throw new Error('Dapps list url is not defined')
+      }
+
+      const response = await fetch(dappsListUrl, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+      })
+
+      const result = await response.json()
+      try {
+        const categoriesById: { [id: string]: CategoryWithDapps } = {}
+        result.categories.forEach((cat: any) => {
+          categoriesById[cat.id] = {
+            id: cat.id,
+            name: cat.name,
+            fontColor: cat.fontColor,
+            backgroundColor: cat.backgroundColor,
+            dapps: [],
+          }
+        })
+        result.applications.forEach((app: any) => {
+          categoriesById[app.categoryId].dapps.push({
+            id: app.id,
+            categoryId: app.categoryId,
+            name: app.name,
+            iconUrl: app.logoUrl,
+            description: app.description,
+            dappUrl: app.url,
+          })
+        })
+
+        return Object.values(categoriesById)
+      } catch (error) {
+        Logger.error(
+          TAG,
+          `There was an error while parsing response: ${JSON.stringify(result)}`,
+          error as Error
+        )
+        throw Error(`There was an error while parsing response: ${(error as Error)?.message}`)
+      }
+    },
+    [],
+    {
+      onSuccess: (result) => {
+        Logger.debug(TAG, `fetch onSuccess: ${JSON.stringify(result)}`)
+      },
+      onError: (error) => {
+        Logger.error(TAG, 'fetch onError', error as Error)
+      },
+    }
+  )
+
+  const onCloseBottomSheet = () => {
+    setBottomSheetVisible(false)
+  }
+
+  const onPressHelp = () => {
+    setHelpDialogVisible(true)
+  }
+
+  const onPressNavigationButton = () => {
+    if (!dappSelected) {
+      // Should never happen
+      Logger.warn(TAG, 'Internal error. There was no dapp selected')
```

Comment:
Should this be `Logger.error` instead? I'm sorry, just now saw it :(

Reply:
Sure